Sources/Sentry/SentryCoreDataTracker.m

```diff
@@ -146,8 +146,9 @@ - (NSString *)descriptionForOperations:
 {
     NSMutableDictionary<NSString *, NSNumber *> *result = [NSMutableDictionary new];
 
-    for (NSManagedObject *item in entities) {
-        NSString *cl = item.entity.name;
+    for (id item in entities) {
+        // entity.name is nullable! fall back to old behavior
+        NSString *cl = ((NSManagedObject *)item).entity.name ?: NSStringFromClass([item class]);
```

Comment:
h: I think we need to use the SwiftDescriptor for a proper swift class name demangling.

```suggestion
        NSString *cl = ((NSManagedObject *)item).entity.name ?: [SwiftDescriptor getObjectClassName:item]);
```